mutiny-core/src/federation.rs

```diff
@@ -76,6 +79,12 @@ const FEDIMINT_OPERATIONS_LIST_MAX: usize = 100;
 
 pub const FEDIMINTS_PREFIX_KEY: &str = "fedimints/";
 
+// Default signet/mainnet federation gateway info
+const SIGNET_GATEWAY: &str = "0256f5ef1d986e9abf559651b7167de28bfd954683cd0f14703be12d1421aedc55";
+const MAINNET_GATEWAY: &str = "025b9f090d3daab012346701f27d1c220d6d290f6b498255cddc492c255532a09d";
+const SIGNET_FEDERATION: &str = "c8d423964c7ad944d30f57359b6e5b260e211dcfdb945140e28d4df51fd572d2";
+const MAINNET_FEDERATION: &str = "c36038cce5a97e3467f03336fa8e7e3410960b81d1865cda2a609f70a8f51efb";
```

Comment:
nit instead of like this might be cleaner if it was an array:

```rust
const PREF_GATEWAYS [(&str, &str); 2] = [
("c8d423964c7ad944d30f57359b6e5b260e211dcfdb945140e28d4df51fd572d2", "0256f5ef1d986e9abf559651b7167de28bfd954683cd0f14703be12d1421aedc55"), // signet
("c36038cce5a97e3467f03336fa8e7e3410960b81d1865cda2a609f70a8f51efb", "025b9f090d3daab012346701f27d1c220d6d290f6b498255cddc492c255532a09d") //mainnet
]
```

this way it is simpler to add more in the future

Reply:
No thanks, I have no intention on using this after fedimint fixes their vetted stuff.